### PR TITLE
fix examples after having enabled property reactivity by default

### PR DESCRIPTION
**Pull request by mariofusco**

### CODE REVIEW

File: drools-examples/src/main/java/org/drools/games/Run.java [hunk 1]
```diff
@@ -15,6 +15,9 @@
 
 package org.drools.games;
 
+import org.kie.api.definition.type.ClassReactive;
+
+@ClassReactive
```
Comment (mariofusco) [inline drools-examples/src/main/java/org/drools/games/Run.java:20]: @tarilabs This is the only actual fix. All other changes are removing redundant @PropertyReactive annotations and replacing retract with delete.

Comment (tarilabs) [reply drools-examples/src/main/java/org/drools/games/Run.java:20]: as `Run` is not conform to Java Beans spec. ok.

**Review by tarilabs — APPROVED** [review]
+1

**Comment by mariofusco** [comment]
ok to test

**Pull request merged**